docs/user-guide/src/bmo/bmo.md

```diff
@@ -0,0 +1,42 @@
+<h1>Bare Metal Operator</h1>
+
+The Bare Metal Operator (BMO) is a custom Kubernetes controller that reconciles BareMetalHost resources using information provided by Ironic and the Cluster API Provider Metal3 (CAPM3) controller. A simplified information flow path is shown below:
+
+```bash
```

Comment:
No need to syntax highlight for bash as this is just text.
````suggestion
```
````

Reply:
Added to silence the linter. Replaced it with text

Reply:
Ah thanks for explaining! Did not know the linter was complaining about this.